plugins/dialog/guest-js/index.ts

```diff
@@ -77,6 +77,72 @@ interface SaveDialogOptions {
   canCreateDirectories?: boolean
 }
 
+/**
+ * Default buttons for a message dialog.
+ *
+ * @since 2.3.0
+ */
+export type MessageDialogDefaultButtons =
+  | 'Ok'
+  | 'OkCancel'
+  | 'YesNo'
+  | 'YesNoCancel'
+
+/**
+ * The Yes, No and Cancel buttons of a message dialog.
+ *
+ * @since 2.3.0
+ */
+export type MessageDialogButtonsYesNoCancel = {
+  /** The Yes button. */
+  yes: string
+  /** The No button. */
+  no: string
+  /** The Cancel button. */
+  cancel: string
+}
+
+/**
+ * The Ok and Cancel buttons of a message dialog.
+ *
+ * @since 2.3.0
+ */
+export type MessageDialogButtonsOkCancel = {
+  /** The Ok button. */
+  ok: string
+  /** The Cancel button. */
+  cancel: string
+}
+
+/**
+ * The Ok button of a message dialog.
+ *
+ * @since 2.3.0
+ */
+export type MessageDialogButtonsOk = {
+  /** The Ok button. */
+  ok: string
+}
```

Comment:
Just a small suggestion, maybe we can change these to something like this, so we reflect the 3 buttons better, also typescript doesn't hint `ok` and `yes` at the same type

```suggestion
export type MessageDialogButtonsYesNoCancel = {
  /** The Yes button. */
  yes: string
  /** The No button. */
  no?: string
  /** The Cancel button. */
  cancel?: string
}
```